pkg/storage/external/external_test.go

```diff
@@ -0,0 +1,22 @@
+package external
+
+import (
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gomods/athens/pkg/storage/compliance"
+	"github.com/gomods/athens/pkg/storage/mem"
+)
+
+func TestExternal(t *testing.T) {
+	strg, err := mem.NewStorage()
+	if err != nil {
+		t.Fatal(err)
+	}
+	handler := NewServer(strg)
+	srv := httptest.NewServer(handler)
+	defer srv.Close()
+	externalStrg := NewClient(srv.URL, nil)
+	clear := strg.(interface{ Clear() error }).Clear
```

Comment:
worth having an explicit named `Clearer` (or something) interface?

Reply:
@arschles this method is only defined/used inside `_test.go`  files and you cannot import it if you define it in a `_test.go` file anyway. So I'd keep it this way, but if we find ourselves doing this casting in other places, then it would def be worth it to define it in a none-test file and use it from within tests similar to our compliance package.

Reply:
What about [here](https://github.com/gomods/athens/pull/1587/files#diff-9ea50d8556888305dac9befb25ed6069R42)?

Reply:
@arschles that is where the method is defined, sorry i meant the interface-casting is only defined/used in _test.go 

The reason that method is defined (and got moved from _test.go) to non-test Go file, is exactly because _test.go files in other packages don't get imported even during tests.

Reply:
I got ya, this is good for me then 👍